lib/tools.py

```diff
@@ -328,9 +335,12 @@ def sort_csv(filename):
     infields = next(csv_reader_infile)
 
     #  write the data to a temporary file and sort it
-    file_temp = open(os.path.normpath("tempfile"), 'wb')
+    file_temp = open(os.path.normpath("tempfile"), 'w')
 
-    csv_writer = csv.writer(file_temp, dialect='excel', escapechar='\\')
+    if os.name == 'nt':
```

Comment:
@goelakash,  I think these two checks should also go if we do not need them, and we can merge. 👍